assets/i18n/en_US.json

```diff
@@ -327,5 +329,34 @@
         "integrationsContributors": "Integrations contributors",
         "cliContributors": "CLI contributors",
         "managerContributors": "Manager contributors"
+    },
+    "installErrorDialog": {
+        "mount_version_mismatch": "Version mismatch",
+        "mount_no_root": "No root access",
+        "mount_missing_installation": "Installation not found",
+
+        "status_failure_blocked": "Installation blocked",
+        "install_failed_verification_failure": "Verification failed",
+        "status_failure_invalid": "Installation invalid",
+        "install_failed_version_downgrade": "Can't downgrade",
+        "status_failure_conflict": "Installation conflict",
+        "status_failure_storage": "Installation storage issue",
+        "status_failure_incompatible": "Installation incompatible",
+        "status_failure_timeout": "Installation timeout",
+        "status_unknown": "Installation failed",
+
+        "mount_version_mismatch_description": "To install by mounting the patched app, the version of the installed app must match the version of the patched app.\n\nInstall the version of the app you are mounting and try again.",
+        "mount_no_root_description": "To install by mounting the patched app, root access must be granted.\n\nGrant root access to ReVanced Manager and try again.",
+        "mount_missing_installation_description": "To install by mounting the patched app, the unpatched app must be installed on this device.\n\nInstall the app and try again.",
+
+        "status_failure_timeout_description": "Try again",
```

Comment:
```suggestion
        "status_failure_timeout_description": "Try again.",
```

Reply:
The description feels too short to convey anything.

![image](https://github.com/ReVanced/revanced-manager/assets/93124920/6cd811f9-072d-43fb-8c33-2005d7c69fa5)

Something like this could work

```suggestion
        "status_failure_timeout_description": "The installation took too long and didn’t finish within the specified period.\n\nWould you like to try again?",
```

![image](https://github.com/ReVanced/revanced-manager/assets/93124920/4f140901-898e-4bc1-b0bd-0aabb2382989)

Reply:
I agree, what would the specified period be to the user though? maybe best to reword that part

Reply:
Hmm, how about:
*  "The installation took too long and didn’t finish within the specified period set by Android. Would you like to try again?"
* "The installation took too long and didn’t finish within the specified period set by your phone. Would you like to try again?"
* "The installation took too long and didn’t finish within the specified period set by your OEM. Would you like to try again?"

Reply:
Maybe remove the specified period thing in its entirety, it just took too long, too long defined by who doesn't matter

Reply:
> Maybe remove the specified period thing in its entirety, it just took too long, too long defined by who doesn't matter

```suggestion
        "status_failure_timeout_description": "The installation took too long to finish.\n\nWould you like to try again?",
```

Feel a little too short in my opinion.